packages/nssm/extra/nssm.ps1

```diff
@@ -1,5 +1,4 @@
-$oldpath = (Get-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH).path
-cd ..\bin
-$nssmlocation = Get-Location
-$newpath = "$oldpath;$nssmlocation"
-Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH -Value $newPath
+ param (
+    [string]$pkg_inst_dpath
+ )
+Copy-Item "$pkg_inst_dpath\bin\nssm.exe" -Destination "c:\Windows" -force
```

Comment:
Why did you choose `c:\Windows`? Isn't this folder for Microsoft's installation of Windows? Can we just put it in `C:\d2iq\dcos`?

Reply:
**nssm** on Windows is an analog of **systemd** on Linux. From our perspective it should a system-wide service too, like **systemd** . In this case the correct place for having it is a system folder, isn't?

Reply:
I'm surprised by your proposal @vadymzagorodko. Nssm was part of `c:/d2iq/dcos from the beginning and therefor I don't see this as a valid request to move it to `c:/windows`.

Reply:
@DominikDary We had a conversation just two days ago (on daily call) that **nssm** isn't a usual package and should be managed in a different way than other packages. Okay, we won't move the file and will use just a symbolic link instead of. As far as I remember it's a common way for DCOS. @OleksandrBielov will add this change.